lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -87,6 +96,24 @@ def script
             fi
           }
 
+          # Strip invalid architectures
+          strip_invalid_archs() {
```

Comment:
Do the swiftmodules actually get used when copied? Fwiw, we don't do anything to the swiftmodules for RealmSwift.